### DISCUSSION TITLE
statusline doesn't change when active buffer is switched

### DISCUSSION THREAD

**Discussion by zoumi** [neoclide/coc.nvim#4422] [Q&A] [answered]
![Snipaste_2022-12-08_21-21-35](https://user-images.githubusercontent.com/5162901/206457101-37a5d9de-6fe3-4880-ae88-24601be9b72c.jpg)
1,open a rust file in tab1
2,open a shell script file in tab2
the status still shows for rust file

**Comment by chemzqm** [comment, marked as answer]
The extension need to use `window.onDidChangeActiveTextEditor` to fix that.

**Reply by fannheyward** [reply]
@zoumi fixed on lastest coc-rust-analyzer.
